src/OrbitBase/FileTest.cpp

```diff
@@ -305,6 +305,40 @@ TEST(File, MoveFile) {
   EXPECT_FALSE(exists_or_error.value());
 }
 
+TEST(File, RemoveFile) {
+  auto tmp_file_or_error = TemporaryFile::Create();
+  ASSERT_THAT(tmp_file_or_error, HasNoError());
+  auto tmp_file = std::move(tmp_file_or_error.value());
+
+  // Windows is weird, for some reason it does not allow to remove opened files.
```

Comment:
That's weird from a Linux perspective, but I've seen the "file is open in another program" error often enough to not find this weird at all. 😀 (Nothing to do here anyway.)

Reply:
In a way this comment could have stayed, it was still helpful. Maybe just "Windows doesn't allow to remove opened files."

Reply:
removing the file it is now part of the test-case we want to make sure RemoveFile on non-existing file returns false.

Reply:
Ah yeah, sorry.